controllers/ReplyQuestionForPresentationInstance.ts

```diff
@@ -0,0 +1,47 @@
+import {Request, Response} from "express";
+import Base = require("./BaseController");
+import Presentation = require("../models/Presentation");
+import PresentationInstance = require("../models/PresentationInstance");
+import User = require("../models/User");
+import PresentationContent = require("../models/PresentationContent");
+
+class ReplyQuestionForPresentationInstance extends Base.BaseController {
+	// Secure this endpoint
+	protected requireLogin = false;
+
+	protected process(req: Request, res: Response, cb: (data: Base.DataContract) => void) {
+
+		var instanceId = req.body.instanceid;
+		var contentId = req.body.contentid;
+		var contentData = JSON.parse(req.body.data);
```

Comment:
You can't submit the form when it's blank. I know it would be best practice to put a check here.